pkg/ocicni/ocicni.go

```diff
@@ -155,9 +155,9 @@ func (plugin *cniNetworkPlugin) monitorConfDir(start *sync.WaitGroup) {
 			logrus.Infof("CNI monitoring event %v", event)
 
 			var defaultDeleted bool
-			createWriteRename := (event.Op&fsnotify.Create == fsnotify.Create ||
+			createWriteRename := event.Op&fsnotify.Create == fsnotify.Create ||
 				event.Op&fsnotify.Write == fsnotify.Write ||
-				event.Op&fsnotify.Rename > 0)
+				event.Op&fsnotify.Rename > 0
```

Comment:
on reflection, shouldn't this be
`event.Op&fsnotify.Rename == fsnotify.Rename`
to match the others

Reply:
yes 👍🏻